wrappers/Julia/CoolProp.jl

```diff
@@ -30,7 +30,7 @@ function PropsSI(FluidName::AbstractString, Output::AbstractString)
 end
 
 # CoolProp::PropsSI(const std::string &, const std::string &, double, const std::string &, double, const std::string&)
-function PropsSI(Output::AbstractString, Name1::AbstractString, Value1::Number, Name2::AbstractString, Value2::Number, Fluid::AbstractString)
+function PropsSI(Output::AbstractString, Name1::AbstractString, Value1::Real, Name2::AbstractString, Value2::Real, Fluid::AbstractString)
```

Comment:
Number include the possibility to have complex input.
Real don't, but allows to send float and integers (with all being converted to the required type for the c library by ccall).
Thus sending floats will generally be faster (no conversion needed), but this is still convenient to have the possibility to send integers.